Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/KlarnaTokenizationViewModel.swift

```diff
@@ -346,15 +357,6 @@ class KlarnaTokenizationViewModel: PaymentMethodTokenizationViewModel {
                         function: #function
                     )
                     
-                    self?.klarnaPaymentSession = res
```

Comment:
don't we need this anymore?

Reply:
Good catch, but it's alright as it's already done on line 117

Reply:
even better 👌